activejob/lib/active_job/test_case.rb

```diff
@@ -1,7 +1,7 @@
 require "active_support/test_case"
 
 module ActiveJob
-  class TestCase < ActiveSupport::TestCase
+  class TestCase < ApplicationTestCase
```

Comment:
I think the idea of `ApplicationTestCase` applies to Rails projects, not the Rails test suite itself.